src/FolioManager.php

```diff
@@ -28,20 +31,24 @@ class FolioManager
      */
     protected ?MatchedView $lastMatchedView = null;
 
+    public function __construct(
```

Comment:
Can you add a simple comment to this method, like we do in any other methods in this project?